Polyester remove ploneorgproducts (#1132)
plone/documentation: develop/plone/getstarted/index.rst

> * `Plomino is a a powerful and flexible web-based application builder for Plone <http://www.plomino.net>`_

Reviewer: ```suggestion
* `Plomino is a a powerful and flexible web-based application builder for Plone <https://github.com/plomino/Plomino/wiki>`_
```

Finding and installing add-on packages
--------------------------------------

Plone add-ons can be found at the `plone.org Add-ons

> <https://plone.org/download/add-ons>`_ page or at the  `PyPI (the Python

Reviewer: ```suggestion
<https://plone.org/download/add-ons>`_ page or at the `PyPI (the Python
```

> Package index) <https://pypi.python.org>`_.

Reviewer: ```suggestion
Package Index) <https://pypi.python.org>`_.
```
